### DISCUSSION TITLE
Params to model

### DISCUSSION THREAD

**Discussion by ulullul** [adonisjs/core#2449] [Help]
How can i pass to model some params? 
i want to get processed my params data from model query  (params maybe request data etc)
example:
MyModel.query().findOrFali(2) // pass params from controller for the query
//and get these in MyModel
class MyModel {
params:['data','data']
}
want to do it without static for one query, for one user

**Comment by RomainLanz** [comment]
Hey @ulullul! 👋 

Could you please rephrase your question?
Maybe add the expected code you will write.
